<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QFrame" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <property name="lineWidth">
   <number>0</number>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>23</y>
     <width>464</width>
     <height>227</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>39</width>
     <height>24</height>
    </size>
   </property>
   <widget class="QWidget" name="H_Channel1">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string/>
    </attribute>
   </widget>
   <widget class="QWidget" name="H_Channel2">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string/>
    </attribute>
   </widget>
   <widget class="QWidget" name="Double_Channel">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string/>
    </attribute>
   </widget>
   <widget class="QWidget" name="L_Channel1">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string/>
    </attribute>
   </widget>
   <widget class="QWidget" name="L_Channel2">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string/>
    </attribute>
   </widget>
   <widget class="QWidget" name="Asset">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string/>
    </attribute>
   </widget>
   <widget class="QWidget" name="Options">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string/>
    </attribute>
    <widget class="QLabel" name="lab_Options">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>21</y>
       <width>191</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
border-image: url(:/widgetphoto/bk/grade2_bk.png);
color:gray;
}</string>
     </property>
     <property name="text">
      <string>参数设置</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="indent">
      <number>7</number>
     </property>
    </widget>
    <widget class="QLabel" name="lab_Debug">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>49</y>
       <width>191</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
border-image: url(:/widgetphoto/bk/grade2_bk.png);
color:gray;
}</string>
     </property>
     <property name="text">
      <string>调试模式</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="indent">
      <number>7</number>
     </property>
    </widget>
    <widget class="QLabel" name="lab_RecWave">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>77</y>
       <width>191</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
border-image: url(:/widgetphoto/bk/grade2_bk.png);
color:gray;
}</string>
     </property>
     <property name="text">
      <string>录波管理</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="indent">
      <number>7</number>
     </property>
    </widget>
    <widget class="QLabel" name="lab_SysInfo">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>105</y>
       <width>191</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
border-image: url(:/widgetphoto/bk/grade2_bk.png);
color:gray;
}</string>
     </property>
     <property name="text">
      <string>系统信息</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <property name="indent">
      <number>7</number>
     </property>
    </widget>
    <widget class="QLabel" name="lab_factory">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>133</y>
       <width>191</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
border-image: url(:/widgetphoto/bk/grade2_bk.png);
color:gray;
}</string>
     </property>
     <property name="text">
      <string>恢复出厂</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="indent">
      <number>7</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="lab_imformation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>253</y>
     <width>301</width>
     <height>17</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color:white;
background:transparent;
}
</string>
   </property>
   <property name="text">
    <string notr="true">TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_freq">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>50</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color:white;}</string>
   </property>
   <property name="text">
    <string notr="true">50Hz</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>7</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color:white;}</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="lab_pwr">
   <property name="geometry">
    <rect>
     <x>412</x>
     <y>14</y>
     <width>25</width>
     <height>9</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lab_pwr_num">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color:white;}</string>
   </property>
   <property name="text">
    <string notr="true">90%</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_wifi">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>25</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color:white;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lab_asset">
   <property name="geometry">
    <rect>
     <x>231</x>
     <y>250</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color:white;}</string>
   </property>
   <property name="text">
    <string>当前设备：未指定</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_temp">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>152</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {color:white;}</string>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <zorder>lab_asset</zorder>
  <zorder>tabWidget</zorder>
  <zorder>lab_imformation</zorder>
  <zorder>lab_freq</zorder>
  <zorder>lab_time</zorder>
  <zorder>lab_pwr</zorder>
  <zorder>lab_pwr_num</zorder>
  <zorder>lab_wifi</zorder>
  <zorder>lab_temp</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
